### DISCUSSION TITLE
What is the gha-runner-scale-set releases

### DISCUSSION THREAD

**Discussion by Moser-ss** [actions/actions-runner-controller#2422] [Questions]
Hey,

When I was looking for an updated version of the runner image, I found out it is stuck in version 2.299.1.
At the same time, we got a new release called `gha-runner-scale-set` and the runner image looks now be hosted in [actions-runner ](https://github.com/actions/runner/pkgs/container/actions-runner)repository

Is it possible to give an update about those new changes, for example, if I need to update the runner image which version should I use?

**Comment by jdagostino9188** [comment]
+1 

Working on deploying self hosted runner scaling solution and don't want to deploy via the old method if its in the process of being replaced.  Docs that exist are not especially robust.
